ekf_filter_node:
  ros__parameters:
    frequency: 20.0
    sensor_timeout: 0.1
    two_d_mode: true
    publish_tf: false

    map_frame: map
    odom_frame: odom
    base_link_frame: base_link
    world_frame: odom

    # Inputs
    odom0: /odom
    odom0_config: [true, true, false,
                   false, false, true,
                   false, false, false,
                   false, false, false,
                   false, false, false]

    odom0_differential: false
    odom0_queue_size: 10

    imu0: /fmu/out/vehicle_imu
    imu0_config: [false, false, false,
                  true,  true,  true,
                  false, false, false,
                  false, false, false,
                  false, false, false]